CONTRIBUTING.md

```diff
@@ -15,7 +15,7 @@ Please note that if you open a bug report and your issue does not follow our tem
 ```bash
 $ git clone github.com:your-username/augur.git
 $ cd augur/
-$ git remote add upstream https://github.com/<your username>/augur.git
```

Comment:
We actually do want users to update their own forks.

Reply:
But wouldn't they use `origin` remote for that? Setting the central repo as upstream would help contributors always be in sync with the latest changes.